ambari-server/src/main/resources/common-services/HIVE/0.12.0.2.0/package/scripts/pre_upgrade.py

```diff
@@ -99,7 +99,7 @@ def convert_tables(self, env):
       hive_kinit_cmd = format("{kinit_path_local} -kt {hive_server2_keytab} {hive_principal}; ")
       Execute(hive_kinit_cmd, user = params.hive_user)
     
```

Comment:
Wasn't there an import in the original PR as well. Do we need it?
https://github.com/apache/ambari/commit/3503896cd57e2a520c350b41e5ec9b62409037a2#diff-24aa47fa4feb8d8a47e6dacf92423639L24

Reply:
I do not see any imports here: https://github.com/apache/ambari/pull/1789/files

Reply:
wrong PR. Its in the other one

Reply:
It was added by the other PR (see #1908 )

Reply:
Thanks for the hint